bundle-size/api.js

```diff
@@ -181,6 +218,54 @@ function extraBundleSizesSummary(otherBundleSizeDeltas, missingBundleSizes) {
   );
 }
 
+/**
+ * Choose the set of all the potential approver teams into a single team.
+ *
+ * Could end up choosing the fallback set defined in the `.env` file, even if
+ * the fallback set is not in the input.
+ *
+ * @param {Array<Array<string>>} allPotentialApproverTeams all the potential
+ *   teams that can approve this pull request.
+ * @param {Logger} log logging function/object.
+ * @param {number} pullRequestId the pull request ID.
```

Comment:
Nit: use prNumber instead of ID; GitHub actually had IDs for pull requests that are present in the payload, so this isn't quite accurate

Reply:
This matches the database schema, so I'd rather not change that at this point...